docs/how-to-use.md

```diff
@@ -627,6 +627,10 @@ master CNI configuration and kubeconfig. when such change detected, the script w
 
     --skip-config-watch
 
+The poll interval of the config changes watcher is configurable with `--multus-config-watch-timer=<duration>`
+
```

Comment:
Let's also provide a link to whatever appropriate go docs about what are valid values for "duration" here. Otherwise, lgtm

Reply:
@dougbtv Thanks. I've updated the description with the go doc link for the `duration`.  Please let me know if any suggestions.

Reply:
@dougbtv Thanks for reviewing the PR. Can we merge the PR or is there something can do to make it better?